sdcm/cluster_aws.py

```diff
@@ -648,6 +648,7 @@ def destroy(self):
         self.log.info('Destroyed')
 
     def start_task_threads(self):
+        self.change_hostname()
```

Comment:
this is not a task thread, should be during init of the node

Reply:
@bentsi Create newc commit, implementing via function in common.py.

Reply:
how did you understand that you need to move this to separate module? Please move it back to the Node class
I meant that `change_hostname` is not a task thread, it shouldn't be started in `start_task_threads` method... (in other words it is a wrong place to put the call to this method)

Reply:
yes, it was clear. The idea was to run change_hostname as earlier as possible, to collect logs with new hostname and for AWSNode only. The start_task_thread was best place, when AWSNode and Basenode Constructor initiated all required variables (new name, remoter).

Reply:
Can we change the hostname by startup script? that's early enough

Reply:
`I think adding the node name to start_script will require to much refactor (from my point of view): Now all instances have same start_script, if put code which set the hostname into it, we will need start_script per instances. Also for that moment we create all instances per cluster type with one call ec2_service.create_instances, otherwise it will require creating one by one.`
reason described before.